checkout-example/index.js

```diff
@@ -40,193 +44,120 @@ app.engine(
     helpers: require("./util/helpers"),
   })
 );
-
 app.set("view engine", "handlebars");
 
-/* ################# API ENDPOINTS ###################### */
-
-// Invoke /sessions endpoint
-app.post("/api/sessions", async (req, res) => {
-
-  try {
-    // unique ref for the transaction
-    const orderRef = uuid();
-    // Allows for gitpod support
-    const localhost = req.get('host');
-    // const isHttps = req.connection.encrypted;
-    const protocol = req.socket.encrypted? 'https' : 'http';
-    // Ideally the data passed here should be computed based on business logic
-    const response = await checkout.PaymentsApi.sessions({
-      amount: { currency: "EUR", value: 10000 }, // value is 100€ in minor units
-      countryCode: "NL",
-      merchantAccount: process.env.ADYEN_MERCHANT_ACCOUNT, // required
-      reference: orderRef, // required: your Payment Reference
-      returnUrl: `${protocol}://${localhost}/handleShopperRedirect?orderRef=${orderRef}`, // set redirect URL required for some payment methods (ie iDEAL)
-      // set lineItems required for some payment methods (ie Klarna)
-      lineItems: [
-        {quantity: 1, amountIncludingTax: 5000 , description: "Sunglasses"},
-        {quantity: 1, amountIncludingTax: 5000 , description: "Headphones"}
-      ] 
-    });
-
-    res.json(response);
-  } catch (err) {
-    console.error(`Error: ${err.message}, error code: ${err.errorCode}`);
-    res.status(err.statusCode).json(err.message);
-  }
+// Middleware to add current year to all responses
+app.use((req, res, next) => {
+  res.locals.currentYear = new Date().getFullYear();
```

Comment:
Suggestion: Remove functionality (remove middleware*) if this is only used for the copyright sign at the bottom.